snap/snapcraft.yaml

```diff
@@ -35,6 +35,7 @@ apps:
       - login-session-observe
       - snap
```

Comment:
Is there anything other than reading icon files that `system-files` is needed for? Otherwise, it seems that `desktop-launch` grants everything required, since it grants `/snap/*/*/**`.

Reply:
Mmm... good point. So maybe we can remove this one... I'll check what does this one.

Reply:
Yes, it seems that it can be safely removed.

Reply:
Removed and rebased.